modules/ROOT/pages/dataweave-formats-flatfile.adoc

```diff
@@ -23,17 +26,108 @@ output application/flatfile schemaPath="src/main/resources/test-data/QBReqRsp.es
 payload
 ----
 
+// CONFIG PROPS GET APPENDED BELOW /////////////////////////////////
+
+
+[[properties]]
+== Configuration Properties
+
+DataWeave supports the following configuration properties for this format.
+
+
+[[reader_properties]]
+=== Reader Properties
+
+This format accepts properties that provide instructions for reading input data.
+
+
+[cols="1,1,1,3a", options="header"]
+|===
+|Parameter |Type |Default|Description
+|`enforceRequires`  |`Boolean`|`false`|Produces an error when set to `true` if a required value is missing.
+
+Valid values are `true` or `false`.
+|`missingValues`  |`String`|`null`|Specifies one of the following fill characters to represent missing values: 
+
+* `none`: Treats all data as values.
+* `spaces`: Interprets a field consisting of only spaces as a missing value. Default for flat file and fixed-width formats. 
+* `zeroes`: Interprets numeric fields consisting of only `0` characters _and_ character fields consisting of only spaces as missing values. 
+* `nulls`: Interprets a field consisting only of `0` bytes as a missing value. Default for COBOL copybook schema.
+
+Valid values are `none` or `spaces` or `zeroes` or `nulls`.
+|`notTruncateDependingOnSubjectNotPresent`  |`Boolean`|`false`|Fills the whole group when the  DEPENDING ON subject is not present.
+
+Valid values are `true` or `false`.
+|`recordParsing`  |`String`|`null`|Specifies the expected type of separation between lines or records: 
```

Comment:
TODO: : recordParsing
default `null` is incorrect. There is no null. 
Correct is `strict`

Creat JIRA for this bc need dev solution.